src/components/Button/PrimaryButton.tsx

```diff
@@ -0,0 +1,151 @@
+import * as React from 'react'
+import styled, { css } from 'styled-components'
+
+import { InjectedProps, withTheme } from '../../hocs/withTheme'
+
+import { hoverable } from '../../hocs/hoverable'
+import { isTouchDevice } from '../../libs/ua'
+
+type Tag = 'button' | 'a'
+type Size = 's'
```

Comment:
I feel uncomfortable that the size is only 's'.

If the value of Size is only 's', props should be 'small' to accept a boolean.

If you keep props as Size, you should accept 'default' or 'small' and set 'default' as defaultValue.
And in style, you should separate the implementation with `.default` and `.small`